tests/cases/whitespace/comments-line

```diff
@@ -0,0 +1,24 @@
+---DESCRIPTION---
+Comment lines must be aligned
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+//Foo
+
+//    Bar
+echo 'baz';    //    qux
+  //      quux
+
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+// Foo
+// Bar
+echo 'baz';    // qux
+  // quux
```

Comment:
Would rather the extra spaces before these two were removed.

Reply:
`Generic.WhiteSpace.ScopeIndent` can be configured to fix the latter.

Reply:
Setting the `exact` value to `true` fixes it, but creates problems elsewhere. For example, indentation of array elements in removed. There is `Generic.Arrays.ArrayIndent` which can be used, but they seem to conflict.

Reply:
Looks problematic to try and resolve, let's not hold up the rest.